src/Drivers/nlpMDSForm_ex6.hpp

```diff
@@ -0,0 +1,651 @@
+#ifndef HIOP_EXAMPLE_EX6
```

Comment:
`HIOP_EXAMPLE_EX6_MDS` to void conflicts with the macro from the sister sparse file

Reply:
Alternatively:
```c++
#pragma once
```